lib/saito/ui/game-playerbox/game-playerbox.js

```diff
@@ -173,18 +251,59 @@ class GamePlayerBox {
       }
     }
 
+    /**
+     * Individually a classname to one of the playerboxes
+     * @param {string} className - name of class
+     * @param {int} player - the player number (according to game), -1 means this player 
+     */
     addClass(className, player = -1){
       let selector = "#player-box-"+this._playerBox(player);
       let box = document.querySelector(selector);
-      if (box)
+      if (box){
         box.classList.add(className);
+      }
     }
 
-  
+    /**
+    * Add a class name to the "graphical" subdivision of each playerbox
+    * @param {string} className - name of class
+    */
+    addGraphicClass(className){
+      let playerBoxes = document.querySelectorAll(".player-box-graphic");
+      for (let hand of playerBoxes){
+        hand.classList.remove(className);
+        hand.classList.add(className);
+      }
+    }
+
+    /**
+     * Adds a "status" class to player-box log of this player so that updateStatus functions
+     * render in the playerbox
+     */ 
+    addStatus(){
+      let div = document.querySelector(`#player-box-log-${this._playerBox(-1)}`);
+      if (div){
+        div.classList.add("status");
+      }
+    }
 
+    /*
+    * Helper class for updating different sections of Player-Box
+    */
+    _updateDiv(selector, html){
+      let div = document.querySelector(selector);
+      if (div){
+        div.innerHTML = html; 
+      }
+      else console.log(selector + " not found");
```

Comment:
If you want to use `console` on purpose here maybe avoid `log` because people confuse it with debugging output. Better use `console.warning` or `console.error`. For just informational purpose you can use `console.info`.